internal/component/loki/source/file/file_test.go

```diff
@@ -359,9 +360,11 @@ func TestDeleteRecreateFile(t *testing.T) {
 	require.NoError(t, f.Close())
 	require.NoError(t, os.Remove(f.Name()))
 
-	// Create a file with the same name
-	f, err = os.Create(filename)
-	require.NoError(t, err)
+	// Create a file with the same name. Use eventually because of Windows FS can deny access if this test runs too fast.
```

Comment:
Are you sure it's because it runs too fast? That would be really odd - surely there's a way to create files quickly on Windows.

In the past I've seen tests fail on Windows due to the files not being closed properly. Could that be the case here?

Reply:
I'm not sure, but on Windows files can be locked and return errors when they were recently accessed. @mattdurham I think has more context.